ledger/light/integration/operations_test.go

```diff
@@ -412,3 +419,11 @@ func RequireError(pl payload.Payload) {
 		panic("expected error")
 	}
 }
+
+func RequireErrorCode(pl payload.Payload, expectedCode uint32) {
```

Comment:
1) why panic on unxpected code?
2) why ignore type assertion error? (just use single return value form if panic is ok on assertion fail)